share/locale/en.json

```diff
@@ -1,165 +1,165 @@
-    "download_page_loaded": "Download page loaded",
-    "other_page_loaded": "Address loaded",
-    "close_on_timeout": "Stopped because timer expired",
-    "closing_automatically": "Stopped because download finished",
-    "timeout_download_still_running": "Waiting for download to complete",
+    "no_filenames": "Add files to OnionShare.",
+    "no_available_port": "Could not start the onion service without a port to do it on.",
+    "download_page_loaded": "OnionShare page loaded",
+    "other_page_loaded": "OnionShare address loaded",
+    "close_on_timeout": "Timer expiry stopped OnionShare",
+    "closing_automatically": "OnionShare completed, now stopped",
+    "timeout_download_still_running": "Awaiting OnionShare completion",
     "large_filesize": "Warning: Sending large files could take hours",
-    "error_tails_invalid_port": "Invalid value, port must be a regular number",
+    "error_tails_invalid_port": "Wrong value, \"Port\" must be a regular number",
     "error_tails_unknown_root": "Unknown error with Tails root process",
     "systray_menu_exit": "Quit",
-    "systray_download_started_title": "OnionShare Download Started",
-    "systray_download_started_message": "A user started downloading your files",
-    "systray_download_completed_title": "OnionShare Download Finished",
-    "systray_download_completed_message": "The user finished downloading your files",
-    "systray_download_canceled_title": "OnionShare Download Canceled",
-    "systray_download_canceled_message": "The user canceled the download",
+    "systray_download_started_title": "OnionSharing",
+    "systray_download_started_message": "Files underway",
+    "systray_download_completed_title": "OnionSharing complete",
+    "systray_download_completed_message": "The user has your files now",
+    "systray_download_canceled_title": "OnionSharing stopped",
+    "systray_download_canceled_message": "The user stopped reception",
     "help_local_only": "Do not attempt to use Tor: For development only",
-    "help_stay_open": "Keep onion service running after download has finished",
-    "help_shutdown_timeout": "Shut down the onion service after N seconds",
-    "help_stealth": "Create stealth onion service (advanced)",
-    "help_receive": "Receive files instead of sending them",
-    "help_debug": "Log application errors to stdout, and log web errors to disk",
-    "help_filename": "List of files or folders to share",
-    "help_config": "Path to a custom JSON config file (optional)",
-    "gui_drag_and_drop": "Drag and drop files and folders\nto start sharing",
+    "help_stay_open": "Keep onion service running after OnionShare is completed",
+    "help_shutdown_timeout": "Close onion service after a given amount of seconds",
+    "help_stealth": "Make onion service stealthy (advanced)",
+    "help_receive": "Set your onion to reception instead of OnionSharing",
+    "help_debug": "Log OnionShare errors to stdout, and log web errors to disk",
```

Comment:
The web errors are sort of still OnionShare errors... but I think this is still probably an improvement so :+1: